### ISSUE TITLE
How to define label during the training procedure

### ISSUE THREAD

**Issue by Ma-yiwei** [naver/deep-image-retrieval#38]
When you do training, how do you define the label? Can you release the text file about your dataset? Thanks a lot

**Comment by liyongqing-1101** [comment]
@Ma-yiwei Have you implemented  your 'train.py'?

**Comment by Ma-yiwei** [comment]
> @Ma-yiwei Have you implemented your 'train.py'?

No, without the label of the whole dataset, I can't accomplish the train code as the author described in the paper.

**Comment by liyongqing-1101** [comment]
I have finished reading this paper today, ready to see the whole codes, but it seems that the author has not released the training code.  :( 
Do you have WeChat, we can learn together.

**Comment by Ma-yiwei** [comment]
> I have finished reading this paper today, ready to see the whole codes, but it seems that the author has not released the training code. :( Do you have WeChat, we can learn together.

ok, I just email you my Wechat. :)
